tests/unit/test-firefox/test.firefox.js

```diff
@@ -545,6 +646,136 @@ describe('firefox', () => {
 
   });
 
+  describe('defaultCreateProfileFinder', () => {
+
+    it('creates a finder', async () => {
+      const FxProfile = {
+        Finder: sinon.spy(() => () => Promise.resolve()),
+      };
+      firefox.defaultCreateProfileFinder({FxProfile});
+      sinon.assert.called(FxProfile.Finder);
+      sinon.assert.calledWith(
+        FxProfile.Finder,
+        sinon.notOk,
+      );
+    });
+
+    it('creates finder based on userDirectoryPath if present', async () => {
+      const FxProfile = {
+        Finder: sinon.spy(() => () => Promise.resolve()),
+      };
+      const userDirectoryPath = '/non/existent/path';
+
+      firefox.defaultCreateProfileFinder({userDirectoryPath, FxProfile});
+
+      sinon.assert.called(FxProfile.Finder);
+      sinon.assert.calledWith(
+        FxProfile.Finder,
+        sinon.match(userDirectoryPath),
+      );
+    });
+
+    it('returns a finder that looks for a default profile', async () => {
```

Comment:
uhm... I think that the description of this test should probably be `'returns a finder that resolves a profile name'` or something like that (instead of 'looks for a default profile').